sys/net/gnrc/transport_layer/tcp/gnrc_tcp_option.c

```diff
@@ -55,7 +55,7 @@ int _option_parse(gnrc_tcp_tcb_t *tcb, tcp_hdr_t *hdr)
                     DEBUG("gnrc_tcp_option.c : _option_parse() : invalid MSS Option length.\n");
                     return -1;
                 }
-                tcb->mss = (option->value[0] << 8) | option->value[1];
+                tcb->mss = (tcp_hdr_opt_value_ptr(option)[0] << 8) | tcp_hdr_opt_value_ptr(option)[1];
```

Comment:
I don't like find the style of "func()[0]" very readable and the calculation is redundant. Please store the Pointer in a variable and use it. I think I would be much clearer to read.